docs/Diagrams.md

```diff
@@ -0,0 +1,18 @@
+# Inner workings of the software
+
+## High-level: BPMN diagram
+
+<img src="https://raw.githubusercontent.com/SergelsOrg/csv2tex/images_and_data/high_level_workflow.bpmn.svg" title="High-level BPMN diagram including user and system actions" style="max-width: 100%;" />
+
+The notation used is [BPMN](https://en.wikipedia.org/wiki/Business_Process_Model_and_Notation).  
+It's semi-formal and can be usually be understood without much background knowledge. 
+Open + edit [the underlying file](images_and_data/high_level_workflow.bpmn) to update this image, e.g. the editor on [this website](https://demo.bpmn.io/) is available free of charge.
+
+## Details: UML Object diagram  
+
+<img src="https://raw.githubusercontent.com/SergelsOrg/csv2tex/images_and_data/uml_object.puml.svg" title="High-level BPMN diagram including user and system actions" style="max-width: 100%;" />
+
+The notation used is [the UML's](https://en.wikipedia.org/wiki/Unified_Modeling_Language) [Object Diagram](https://en.wikipedia.org/wiki/Object_diagram).
+Open + edit [the underlying file](images_and_data/uml_object.puml) to update this image, e.g. the editor on [this website](https://www.plantuml.com/plantuml/uml/) is available free of charge. 
```

Comment:
:x: Ah, sorry, I misremembered in terms of: https://www.tutorialspoint.com/uml/uml_interaction_diagram.htm Collaboration Diagram, this is the one I wanted. 

As the Object diagram has no timing information, I'll switch to a Sequence Diagram soon, I guess

Reply:
Done :heavy_check_mark: